pkg/hook/controller/kubernetes_bindings_controller.go

```diff
@@ -99,6 +100,57 @@ func (c *kubernetesBindingsController) EnableKubernetesBindings() ([]BindingExec
 	return res, nil
 }
 
+func (c *kubernetesBindingsController) UpdateMonitor(monitorId string, kind, apiVersion string) error {
```

Comment:
We will face the problem of executing this method concurrently, won't we?

Reply:
I thought about this situation. It can happen for the hook with different queues but Go hook has only one queue, so there is no way to execute UpdateMonitor with one `monitorId` in parallel.

Reply:
Make sense. It will be a problem if we make shared monitors, but I agree that for now, everything is ok.